book/scripts.md

```diff
@@ -309,14 +309,12 @@ ignore the crash. Web pages shouldn't be able to crash our browser!
 You can implement that like this:
 
 ``` {.python}
-class Tab:
```

Comment:
yikes

Reply:
Yeah, this was kinda ugly. Basically, in Chapter 9 we put the error-handling code in `Tab.load`; in Chapter 12 we want to move it to `run` because then we can use `Task(self.js.run, ...)`. Better put it in its final location from the beginning.

Reply:
Is this an orthogonal fix you found along the way? Or does it benefit the outline in a concrete way?

Reply:
It benefits the outline because moving the error handling into `JSContext.run` requires adding a `script` parameter, meaning that the outline changes (a new parameter appears).